packages/scaffolder/src/error.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
+
+import chalk from 'chalk';
+import { logger } from './logger';
+
+/**
+ * Exit error display message.
+ */
+export default function handleError(message: string): never {
+  logger().error(chalk?.red(message)); // eslint-disable-line no-console
```

Comment:
Maybe disable no-console in eslint for this whole package? Since it's a CLI console messages are expected.

Reply:
I actually opted to just pipe everything through Winston (the logger) to prevent the need to disable it.